src/mame/drivers/jalmah.cpp

```diff
@@ -126,18 +126,22 @@ OSC:    12.000MHz
 class jalmah_state : public driver_device
 {
 public:
-	jalmah_state(const machine_config &mconfig, device_type type, const char *tag)
-		: driver_device(mconfig, type, tag),
+	jalmah_state(const machine_config &mconfig, device_type type, const char *tag) :
+		driver_device(mconfig, type, tag),
 		m_palette(*this, "palette"),
 		m_tmap(*this, "scroll%u", 0),
 		m_jm_shared_ram(*this, "jshared_ram"),
 		m_jm_mcu_code(*this, "jmcu_code"),
-		m_maincpu(*this, "maincpu")
-		{ }
+		m_maincpu(*this, "maincpu"),
+		m_prirom(*this, "prirom"),
+		m_okibank(*this, "okibank"),
+		m_key_io(*this, "KEY%u", 0U),
+		m_system_io(*this, "SYSTEM"),
+		m_dsw_io(*this, "DSW") { }
```

Comment:
Please don't move the braces to the end of the line when the initialiser list isn't all on one line like this.